doc/configuration.rst

```diff
@@ -1098,6 +1099,46 @@ optimize less but speed up the build time you could do::
 
 See ``$ optipng --help`` for a complete list of options.
 
+.. _image_srcset:
+
+Multi-resolution images
+=======================
+
+Web browsers allow a ``srcset`` parameter to the ``<img>`` tag that 
+allows the browser to support `responsive resolution images 
+<https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images>`__
+for hi-dpi/retina displays. Sphinx Gallery supports this via the ``image_srcset`` parameter::
+
+    sphinx_gallery_conf = {
+        ...
+        'image_srcset': ["", "2x"],
```

Comment:
Outdated, right?
```suggestion
        'image_srcset': ["2x"],
```

Reply:
Oops, yes, that change didn't get pushed...